src/hooks/useAuth.ts

```diff
@@ -0,0 +1,6 @@
+import { useSelector } from 'react-redux';
+import { RootState } from 'store/store';
+
+export const useAuth = () => {
+  return useSelector((state: RootState) => state.user);
+};
```

Comment:
put it in selectors `(state: RootState) => state.user`
use useAppSelector don't create unnecessary hooks